### ISSUE TITLE
Redis - wrong number of arguments for 'hset'

### ISSUE THREAD

**Issue by LukasJK** [hudy9x/namviek#177]
**Describe the bug**
When signing in after seeding the default user, yarn:backend shows the error
```
 🥝 POST: /auth/sign-in
service get user by id error ReplyError: ERR wrong number of arguments for 'hset' command
    at parseError (/home/{...}/namviek/node_modules/redis-parser/lib/parser.js:179:12)
    at parseType (/home/{...}/namviek/node_modules/redis-parser/lib/parser.js:302:14) {
  command: { name: 'hset', args: [ 'USER_admin@google.com', '' ] }
}
```

Redis 7.2.4

**Comment by hudy9x** [comment]
@LukasJK thanks for your report, i'll fix this

**Comment by hudy9x** [comment]
this fixed by #186
